### DISCUSSION TITLE
Multiple panels based on contexts w/ shared nav elements.

### DISCUSSION THREAD

**Discussion by patrickcurl** [filamentphp/filament#2082] [Ideas]
I've created a fork to try and do this on my own.. I've renamed it to Dasher, so it's differentiated. Very unfinished at the moment... 
github.com/zvive/dasher

I'm thinking your config would have a list of dashboards, so the same config but nested inside an array for each dash. 

filament.dashboards = [ 'admin' => [...], 'shop' => [...], 'projects' => [...]]

Then in those you can choose namespaces/paths for everything, another nice to have would be remove any app_path's and use a full path and base_path so that Module support in src/Domain/ instead of app/Modules works well.. 

Looking through the code, I think you could do this w/ a separate Service Provider that just loops over the dashboards, and then figuring out how to share components, like -- say you have a dropdown on the top right to choose a context/view, for instance: kanban boards, shops, admin, employee dashboard, etc.. depending on what the user has 'privileges' to see. 

I think this is a brilliant package, it just seems limited that you have to have everything under admin/ and can't create separate views/panels w/ their own roots, etc. Like you could have all your classes in App\Dashboards\Admin, App\Dashboards\Shop, etc...

**Comment by ryangjchandler** [comment]
This sort of thing would involve major changes in Filament's architecture. We're not saying never though, I could definitely see a few usecases for this myself.

**Comment by invaders-xx** [comment]
I will definitely enjoy that too!

**Comment by danharrin** [comment]
It would affect the major elements of how the admin panel is registered and routed, but thankfully very little of the actual work that went into these packages would change.

I'd still want to keep the same easy onboarding experience that we have now, i.e. don't force users to create a base admin panel when they first set up Filament.
